site/pages/search.html

```diff
@@ -0,0 +1,337 @@
+    line-height: 1.58;
+  }
+
+  p:last-child {
+    margin-bottom: 0px;
+  }
+
+  .url {
+    font-weight: normal;
+    margin-bottom: 10px;
+  }
+
+  #pagination {
+    width: 100%
+  }
+
+  #pagination>* {
+    margin: auto;
+    display: flex;
+  }
+
+  #pagination ul {
+    margin-left: auto;
+    margin-right: auto;
+    padding: 0px;
+  }
+
+  .ais-pagination--item__disabled {
+    visibility: visible;
+    color: #efefef;
+  }
+
+  .ais-pagination--item {
+    display: inline-block;
+    padding: 3px;
+  }
+
+  li.ais-pagination--item.ais-pagination--item__page.ais-pagination--item__active>* {
+    text-decoration: none;
+    color: #283891;
+  }
+
+  .search-hits h5 b {
+    font-weight: bold;
+  }
+
+  #nhits {
+    color: #444;
+    margin: 0px 0px 50px 0px;
+  }
+
+  @media (max-width: 767px) {
+    #hits {
+      margin-left: 15px;
+      max-width: 100%;
+    }
+
+    #nhits {
+      margin: 12.5px 0px 25px 0px;
+    }
+  }
+</style>
+
+{% endblock %}
+{% block content %}
+
+<div id="hero" style='flex-direction:column'>
+  <div id='hits'></div>
+</div>
+<div id='pagination'></div>
+<script src="https://cdn.jsdelivr.net/npm/instantsearch.js@2.8.1"></script>
+<script src='https://cdnjs.cloudflare.com/ajax/libs/algoliasearch/4.2.0/algoliasearch-lite.umd.js'></script>
+<script>
+  (function () {
+    const client = algoliasearch('BH4D9OD16A', 'd2dee24912091336c40033044c9bac58');
+
+    if (location.search) {
+      const cachedSearchInput = document.getElementById("search")
+      const searchTerms = new URLSearchParams(location.search.slice(1));
+
+      if(searchTerms.get("query")) {
+        cachedSearchInput.value = searchTerms.get("query");
+      }
+    } else {
+      cachedSearchInput.value = "";
+    }
+
+
+    // required for url querystring-based search on refresh
+    const renderSearchBox = (renderOptions, isFirstRender) => {};
+    const customSearchBox = instantsearch.connectors.connectSearchBox(renderSearchBox);
+
+    const search = instantsearch({
+      indexName: 'hail_is',
+      searchClient: client,
+      routing: {
+        router: instantsearch.routers.history({
+          windowTitle({ category, query }) {
+            const queryTitle = query ? `Results for "${query}"` : 'Search';
+            return queryTitle;
+          },
+
+          createURL({ qsModule, routeState, location }) {
+            const searchTerms = new URLSearchParams(location.search.slice(1));
+            const query = decodeURIComponent(searchTerms.get("query"));
+
+            const queryParameters = {};
+
+            if (query) {
+              queryParameters.query = encodeURIComponent(query);
+            }
+            if (routeState.page !== 1) {
+              queryParameters.page = routeState.page;
+            }
+
+            const queryString = qsModule.stringify(queryParameters, {
+              addQueryPrefix: true,
+              arrayFormat: 'repeat'
+            });
+
+            return `/search.html${queryString}`;
```

Comment:
Noticed this while I was looking at the dev deployment. If we want to unbreak search in dev deployment, this should be a releative URL path, the leading slash is causing the path to be replaced, not joined to.

Reply:
I'll fix but I think search is deeply broken in dev anyway because all the links point back to hail.is.